### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by wacii**
There were inconsistent helpers in the `<ControlledRadioYesNo>` test that I wanted to investigate before fixing.

Also some improvements to the snapshot testing. I made sure each has a meaningful name, broke up tests that contained multiple snapshot comparisons, and converted the ones that were basically smoke tests to test for errors explicitly using `.not.toThrow()`.

The intent of these tests should be clearer now, a general improvement but it also makes it easier to convert them to proper assertions, if that is what people want to do.

This only involved the tests I wrote.

### CODE REVIEW

File: src/test/components/GraphTimeButtons.test.js [hunk 1]
```diff
@@ -11,7 +11,12 @@ describe('<GraphTimeButtons>', () => {
     expect(wrapper.find('#Yearly').exists()).toBe(true);
   });
 
-  it('matches snapshot', () => {
-    expect(mount(<GraphTimeButtons />)).toMatchSnapshot();
+  it('renders with provided props', () => {
+    const props = {
+      activeID:   'not sure what this should be',
```
Comment (knod) [inline src/test/components/GraphTimeButtons.test.js]: iirc, that's supposed to actually be a time interval value - weekly, monthly, or yearly.

Comment (wacii) [reply src/test/components/GraphTimeButtons.test.js]: Whoops. Meant to look that up.

File: src/test/components/GraphTimeButtons.test.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ describe('<GraphTimeButtons>', () => {
 
   it('renders with provided props', () => {
     const props = {
-      activeID:   'not sure what this should be',
+      activeID:   'monthly',
```
Comment (knod) [inline src/test/components/GraphTimeButtons.test.js]: Hmm, it might need to be 'Monthly'.

Comment (knod) [reply src/test/components/GraphTimeButtons.test.js]: @todo Change `activeID` to `timeInterval`

**Review by knod — APPROVED** [review]
That looks good to me! I have questions about 'why this and that', but that's just curiosity and we can chat about it over candy corn and apples or whatever.

**Review by turnerhayes — APPROVED** [review]
Looks good to me!

**Pull request merged**